src/protocols/device_path_to_text.rs

```diff
@@ -0,0 +1,30 @@
+//! Device Path to Text Protocol
+//!
+//! Convert device nodes and paths to text.
+
+pub const PROTOCOL_GUID: crate::base::Guid = crate::base::Guid::from_fields(
+    0x8b843e20,
+    0x8132,
+    0x4852,
+    0x90,
+    0xcc,
+    &[0x55, 0x1a, 0x4e, 0x4a, 0x7f, 0x1c],
+);
+
+pub type DevicePathToTextNode = eficall! {fn(
+    *const crate::protocols::device_path::Protocol,
+    crate::base::Boolean,
+    crate::base::Boolean,
+) -> *const crate::base::Char16};
```

Comment:
This should be `*mut`, right? The value is allocated and free to the caller to use in any way they wish. They also have to release it. And the spec does not have any `const` annotation either. Same for the function below.